src/core/constants.js

```diff
@@ -60,7 +61,8 @@ export type AddonTypeType =
   | typeof ADDON_TYPE_LANG
   | typeof ADDON_TYPE_OPENSEARCH
   | typeof ADDON_TYPE_STATIC_THEME
-  | typeof ADDON_TYPE_THEME;
+  | typeof ADDON_TYPE_THEME
+  | typeof ADDON_TYPE_THEMES_FILTER;
```

Comment:
Why do we need to treat `ADDON_TYPE_THEMES_FILTER` as an add-on type? To me, it is only a filter for the API, not a type.

Reply:
good catch, thanks